### PR TITLE
Fix handling of <include> elements in documentation

### LINKED ISSUE #1603: SA1604, SA1615 do not support the <include> tag
Consider the following files:

**Class.cs**:

```csharp
// <copyright file="Class1.cs" company="PlaceholderCompany">
// Copyright (c) PlaceholderCompany. All rights reserved.
// </copyright>

namespace ClassLibrary3
{
    using System.Threading.Tasks;

    /// <summary>
    /// Sample class.
    /// </summary>
    public class Class1
    {
        /// <include file='Class1.xml' path='/Class1/Foo/*'/>
        public async Task Foo()
        {
            await Task.Delay(0);
        }
    }
}
```

**Class1.xml**:

```xml
<?xml version="1.0" encoding="utf-8" ?>
<Class1>
  <Foo>
    <summary>
      Sample method.
    </summary>
    <returns>
      A <see cref="Task"/> representing the asynchronous operation.
    </returns>
  </Foo>
</Class1>
```

The method `Foo` is documented and the C# compiler correctly generates the documentation for this method.

However, StyleCop reports:

```
Warning	SA1604	Element documentation must have summary	ClassLibrary3	Class1.cs	15
Warning	SA1615	Element return value must be documented	ClassLibrary3	Class1.cs	
```

Classic StyleCop did support XML `<include>` statements

### PR DESCRIPTION
**Pull request by sharwell**
This change only updates SA1604, SA1606, and SA1615 to properly handle
these comments.

Fixes #1603

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1604UnitTests.cs [hunk 1]
```diff
@@ -6,14 +6,17 @@ namespace StyleCop.Analyzers.Test.DocumentationRules
...
     using Microsoft.CodeAnalysis.Diagnostics;
     using StyleCop.Analyzers.DocumentationRules;
+    using StyleCop.Analyzers.Test.Helpers;
     using TestHelper;
     using Xunit;
 
     /// <summary>
     /// This class contains unit tests for <see cref="SA1604ElementDocumentationMustHaveSummary"/>.
     /// </summary>
+    /// <include file='../../StyleCop.Analyzers.ruleset' path='/Class1/Foo/*'/>
```
Comment (pdelvo) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1604UnitTests.cs]: Is this supposed to be here?

Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1604UnitTests.cs]: It is not.

**Comment by pdelvo** [comment]
Otherwise :+1:

**Comment by vweijsters** [comment]
:+1:

**Pull request merged**